apis/voters.js

```diff
@@ -163,12 +164,19 @@ router.post('/verifyTx', async (req, res, next) => {
             if (error) {
                 if (action === 'vote') {
                     web3.eth.getBalance(signedAddress, function (e, balance) {
```

Comment:
Need try catch this function:  web3.eth.getBalance(signedAddress, function (e, balance)